tests/banks.py

```diff
@@ -37,6 +37,41 @@ def test_success_fetch_accounts(requests_mock):
     assert response == result
 
 
+def test_success_fetch_accounts_on_page_2(requests_mock):
+    results = [
+        {
+            "id": "5a8c7990-393a-4299-ae92-2f096a2c7f43",
+            "created_date": "2020-10-08T02:18:07.346849Z",
+            "modified_date": "2020-10-08T02:18:07.346914Z",
+            "account_number": "a37e2836805975f334108b55523634c995bd2a4db610062f404510617e83126f",
+            "trust": "0.00",
+        },
+        {
+            "id": "2682963f-06b1-47d7-a2e1-1f8ec6ae98dc",
+            "created_date": "2020-10-08T02:39:44.071810Z",
+            "modified_date": "2020-10-08T02:39:44.071853Z",
+            "account_number": "cc8fb4ebbd2b9a98a767e801ac2b0d296ced88b5d3b7d6d6e12e1d2d7635d724",
+            "trust": "0.00",
+        },
+    ]
+
+    address = "10.2.3.4"
+    url = f"http://{address}:80/accounts"
+
+    payload = {
+        "count": 4,
+        "next": f"{url}?limit=2&offset=4",
+        "previous": f"{url}?limit=2",
+        "results": results,
```

Comment:
Just to make it close to reality. If `count` is 4, `next` would be None, You can set `count` > 4 or set `next` to None.

Reply:
Thanks for catching this, the count should have been `6`.